pkg/server/app/app.go

```diff
@@ -328,6 +330,33 @@ func checkForProtectedEnvVars(evsNames []string) error {
 	return nil
 }
 
+func (ops *AppOperations) List(user *storage.User) ([]*List, error) {
+	teams, err := ops.tops.ListByUser(user.Email)
+	if err != nil {
+		log.Errorf("ErrNotFound")
+	}
+	lists := make([]*List, 0)
+	for _, team := range teams {
+		NameApps, err := ops.kops.FindAppByLabel(team.Name)
```

Comment:
All the local variables should start with a lowercase. And by the way, I'd name this variable `appNames` instead.